tests/functional/verification/issues/00131.py

```diff
@@ -0,0 +1,2 @@
+def test()-> None:
```

Comment:
I don't think I understand the purpose of this test file.

Reply:
All Python methods generate Viper methods with a goto and a label statement. So the presence of one empty Python function/method is enough to trigger the faulty message when performing the check transitively. Any Python function or method would be enough, this is just the minimal.

Reply:
File removed.

Reply:
Also fixed conflict during merge with master.